### PR TITLE
winspool support added & added gdi32 functions

### PR DESCRIPTION
**Pull request by skdltmxn**
this commit adds winspool support

### CODE REVIEW

File: tests/winspool.rs [hunk 1]
```diff
@@ -0,0 +1,142 @@
+// Copyright © 2015, Jordan Miner
```
Comment (retep998) [inline tests/winspool.rs]: I think you got the wrong copyright here.

**Comment by skdltmxn** [comment]
oops, i fixed it :P

**Comment by retep998** [comment]
The appveyor build is failing because some of the functions don't exist in the bundled libwinspool.a in Rust. Do you think you could add a `#[cfg(target_env = "msvc")]` to those functions in the test so they're only tested on msvc?

**Comment by skdltmxn** [comment]
ok, i separated them, thanks!

**Comment by retep998** [comment]
Do you think you could squash the commits together?

**Comment by skdltmxn** [comment]
yes, i think i squashed them now

**Comment by skdltmxn** [comment]
There were mistakes regarding to coding convention (i missed commas for multiline functions).
Fixed and squashed all including gdi32 additions.

File: src/wingdi.rs [hunk 2]
```diff
@@ -245,3 +245,124 @@ pub type LPABCFLOAT = *mut ABCFLOAT;
...
+
+#[repr(C)] #[derive(Clone, Copy, Debug)]
+pub struct LOGPEN {
+    pub lopnStyle: ::UINT,
+    pub lopnWidth: ::POINT,
+    pub lopnColor: ::COLORREF
+}
+pub type PLOGPEN = *mut LOGPEN;
+pub type NPLOGPEN = *mut LOGPEN;
+pub type LPLOGPEN = *mut LOGPEN;
```
Comment (retep998) [inline src/wingdi.rs]: Missing end of file newline.

File: src/lib.rs [hunk 3]
```diff
@@ -1911,6 +1911,41 @@ pub type PPIXELFORMATDESCRIPTOR = *mut PIXELFORMATDESCRIPTOR;
 pub type LPPIXELFORMATDESCRIPTOR = *mut PIXELFORMATDESCRIPTOR;
 
 //-------------------------------------------------------------------------------------------------
+// winspool.h
```
Comment (retep998) [inline src/lib.rs]: Could you split this off into `winspool.rs`?

**Comment by skdltmxn** [comment]
new line added, and splitted winspool.rs.
sorry for lots of mistakes..

**Pull request merged**